pkg/sharky/store.go

```diff
@@ -158,7 +161,11 @@ func (s *Store) Write(ctx context.Context, data []byte) (loc Location, err error
 
 	select {
 	case e := <-c:
-		if e.err != nil {
+		if e.err == nil {
+			size := float64(int(e.loc.Length) / s.datasize)
```

Comment:
so each write will be length datasize ideally. This means this is always going to be 1.

Reply:
Done.